src/citrine/informatics/scores.py

```diff
@@ -17,6 +17,9 @@ class Score(PolymorphicSerializable['Score']):
 
     """
 
+    _name = properties.String('name')
+    _description = properties.String('description')
```

Comment:
If they're only needed for deserialization, we can mark them as `serializable=False` so we won't even send anything to the backend when POSTed.

Reply:
They're only needed for serialization, so I marked `deserializable=False`